### PR TITLE
Add badge for Coveralls

### PR DESCRIPTION
**Pull request by erikdesjardins**
27% coverage seems a *little* high, I'll look into that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Reddit Enhancement Suite
 
 [![Build Status](https://travis-ci.org/honestbleeps/Reddit-Enhancement-Suite.svg?branch=master)](https://travis-ci.org/honestbleeps/Reddit-Enhancement-Suite)
+[![Build status](https://ci.appveyor.com/api/projects/status/ah6r3ij1yqtkc0ad/branch/master?svg=true)](https://ci.appveyor.com/project/erikdesjardins/reddit-enhancement-suite-md1r4/branch/master)
```
Comment (erikdesjardins) [inline README.md]: Interesting, seems that Appveyor projects are per-user, not per-repo. It's definitely building this repo though: https://ci.appveyor.com/project/erikdesjardins/reddit-enhancement-suite-md1r4/build/1/job/3v0yu95bd1o25ou3#L2

**Comment by erikdesjardins** [comment]
6% coverage is much more like it!

**Comment by Ajedi32** [comment]
:+1: For this. Test coverage for RES isn't great, and I think making that problem more visible is a good first step to fixing it.

**Comment by Ajedi32** [comment]
Are you sure this is working... the coverage reports seem to be showing a lot of lines that actually *are* tested as uncovered: https://coveralls.io/builds/5630299/source?filename=lib%2Fcore%2Futils.js#L1419

**Comment by erikdesjardins** [comment]
It did when this PR was merged (18.19% coverage in utils.js): https://coveralls.io/builds/5606982/source?filename=lib%2Fcore%2Futils.js

Seems like ignoring `dist` and `lib/vendor` messed it up, which is a bit absurd.
https://coveralls.io/builds/5630077/source?filename=lib%2Fcore%2Futils.js
I'll look into it

**Pull request merged**